components/UI/Drawer/components/channels/babala/FeedChannelBabala.tsx

```diff
@@ -1,32 +1,109 @@
 import Typography from "@mui/material/Typography";
+import { Check } from "@mui/icons-material";
+
 import { FeedChannelBabalaProps } from "../../types";
+import { Link, Popover } from "@mui/material";
+import { useState } from "react";
 
 const FeedChannelBabala = ({
   full_text,
   extra_parameters,
+  reason,
 }: FeedChannelBabalaProps) => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState<any>(false);
+
   return (
     <>
       <div style={styles.container}>
         <div style={styles.user}></div>
+        <div style={styles.logo_container}>
+          <Typography style={styles.logo}>Babala</Typography>
+          {!!extra_parameters?.status && (
+            <Link
+              onMouseEnter={() => setIsPopoverOpen(true)}
+              onMouseLeave={() => setIsPopoverOpen(false)}
+            >
+              <Check style={styles.confirmation} />
+            </Link>
+          )}
+          {!!reason && (
+            <Typography style={styles.reason}>
+              {reason.toLowerCase()}
+            </Typography>
+          )}
+        </div>
         <Typography style={styles.fullText}>{full_text}</Typography>
-        <Typography style={styles.fullText}>
-          {extra_parameters?.status}
-        </Typography>
-        <Typography style={styles.confirmation}>
-          {extra_parameters?.manual_confirmation}
-        </Typography>
       </div>
+      <Popover
+        anchorReference="anchorEl"
+        open={isPopoverOpen}
+        anchorOrigin={{
+          vertical: "bottom",
+          horizontal: "left",
+        }}
+        transformOrigin={{
+          vertical: "top",
+          horizontal: "right",
+        }}
+        sx={{
+          background: "#00000000",
+          pointerEvents: "none",
+          zIndex: 10000000,
+        }}
+        onClose={() => setIsPopoverOpen(false)}
+      >
+        <Typography sx={{ padding: "10px", width: "450px" }}>
+          Veri doğrulanmış.
+        </Typography>
+      </Popover>
     </>
   );
 };
+
 // TODO#642: Babala channelından gelen veriler için gösterilecek tasarım
 const styles = {
+  reason: {
+    backgroundColor: "#F05454",
+    fontWeight: 800,
+    color: "white",
+    fontFamily:
+      '-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif',
+    fontSize: 12,
+    width: "fit-content",
+    borderRadius: 6,
+    paddingRight: 10,
```

Comment:
Hello! It would be a good improvement if this padding lines can written as
`padding: "5px 10px"`. Instead of 4, we'll have 1 line 

But if the PR is urgent, then don't mind. It's just about improving code quality